x-pack/plugins/security_solution/public/management/cypress/support/e2e.ts

```diff
@@ -31,12 +31,12 @@ declare global {
   // eslint-disable-next-line @typescript-eslint/no-namespace
   namespace Cypress {
     interface Chainable {
-      getBySel(...args: Parameters<Cypress.Chainable['get']>): Chainable<JQuery<HTMLElement>>;
+      getByTestId(...args: Parameters<Cypress.Chainable['get']>): Chainable<JQuery<HTMLElement>>;
     }
   }
 }
 
-Cypress.Commands.add('getBySel', (selector, ...args) =>
+Cypress.Commands.add('getByTestId', (selector, ...args) =>
```

Comment:
I know it's just a tiny change in naming, but do we actually get by TestId here or TestSubj as everywhere else ?

Reply:
What's the return type of this method?

Some suggestions:

- I think it should be a single `Element` (perhaps defined by a generic)
- if not. found, this method should `throw`
- if multiples are found, an error should be thrown (we can create `getAllByTestSubj()` if needed)


FYI: The reason I tend to suggest we use "common" methods across test tools is so that the API surface remains small and consistent across them. It just makes it for a better DX (IMO).

Reply:
the default behavior of Cypress `get` method is to return one or more elements, so I don't see the benefits of having 2 methods for getting the elements (one for a single return `getByTestSub()` and one for `getAllByTestSubj()`), because commands are chainable you often define filters as `.getByTestSubj().contains(...)`, so I'm not sure how it supposed to work
https://glebbahmutov.com/blog/cypress-v12/